rfcs/0000-sudt-info/0000-sudt info.md

```diff
@@ -0,0 +1,184 @@
+---
+Number: "0000"
+Category: Standards Track
+Status: Proposal
+Author: Xiang Cheng
+Organization: Lay2dev
+Created: 2021-12-25
+---
+
+# sudt info
```

Comment:
```suggestion
# SUDT Info
```

Should use correct upper/down case in title.

Reply:
The cases in following occurrences of 'SUDT info' is not consistent, sometimes it's 'sudt info', sometimes it's 'sudt Info'. Should use either 'SUDT Info' or 'SUDT info'.